pom.xml

```diff
@@ -206,7 +206,7 @@
                 <plugin>
                     <groupId>org.openapitools</groupId>
                     <artifactId>openapi-generator-maven-plugin</artifactId>
-                    <version>4.1.2</version>
+                    <version>4.1.3</version>
```

Comment:
I regenerated the IngestAPI to confirm that our custom interface didn't get out of sync:+1: